src/overview/introduction.md

```diff
@@ -24,7 +24,7 @@ Fortunately, Google met the same problem and had already published some papers t
 
 Following is the architecture of TiKV:
 
-![Architecture](architecture.png)
+![Architecture](../overview/architecture.png)
```

Comment:
oh, strange 😭 

can `./architecture.png` work?

Reply:
Seems not. It tries to find "/deep-dive-tikv/book/architecture.png".

Reply:
Only for this image. All others do not need this. It's only because it's a special case, the first page.